CHANGELOG.md

```diff
@@ -1,3 +1,124 @@
+## Draft for 2.0.0
+
+## BREAKING CHANGES
+
+This release introduce several breaking changes and requires to migrate some parts.
+
+### Dropped some connectors and introduce a new connector for `ApplicationState`
+
+The `@commercetools-frontend/application-shell-connectors` used to contain some connectors that have now been **removed**:
+
+- **configuration**: `injectConfiguration` and `<ConfigurationConsumer>`
+- **project-data-locale**: `withProjectDataLocale` and `<GetProjectDataLocale>`
+- **user-permissions**: `withUserPermissions` and `<GetUserPermissions>`
+- **user-time-zone**: `withUserTimeZone` and `<GetUserTimeZone>`
+
+As a replacement, we now have a single component `<GetApplicationState>` and `withApplicationState` that provide all the necessary "global" information about the `user`, `project` and application `environment`.
```

Comment:
maybe reword this to say that we have a single component, and a single hoc? It's sort of confusing to say we have a single component and then list two.

Reply:
nitpick: `provide` -> provides